src/fam-api/fam_ops_libfabric.cpp

```diff
@@ -63,13 +63,15 @@ Fam_Ops_Libfabric::~Fam_Ops_Libfabric() {
 }
 
 Fam_Ops_Libfabric::Fam_Ops_Libfabric(bool source, const char *libfabricProvider,
+		                     const char *pe_if_device,
```

Comment:
I guess this code is common for memory server and pe. You can rename the variable to *if_device or if_device_str.